stable/kommander/charts/kommander-thanos/requirements.yaml

```diff
@@ -1,4 +1,4 @@
 dependencies:
   - name: thanos
-    version: 0.3.9
+    version: 0.3.20
```

Comment:
Just double checking, we are jumping 11 versions yes? Have we done a spot check test to make sure that things look solid when doing this upgrade?

Reply:
Yep upgrades all look good! Not a major version jump and I checked the diff, no really huge changes